website/views/shared/tasks/task_view/index.jade

```diff
@@ -37,7 +37,8 @@
     label(for='box-{{::obj._id}}_{{::task._id}}')
 
 // main content
-.task-text(ng-dblclick='task._editing ? saveTask(task) : editTask(task)')
+// don't allow a double click to edit if the object is locked
+.task-text(ng-dblclick='obj._locked ? {} : task._editing ? saveTask(task) : editTask(task, obj)')
```

Comment:
So that's what I tried to do initially but I don't know what the obj is and the controller doesn't appear to have access to it somehow? The method signature is already (task, user) and we're only sending task so adding obj didn't seem ideal. Do you know what this obj is?